records_mover/records/sources/fileobjs.py

```diff
@@ -154,14 +155,28 @@ def to_dataframes_source(self,
             else:
                 chunksize = int(entries_per_chunk / num_fields)
 
+            def fix_integer_columns(dfs: Iterator['Dataframe']) -> Iterator['Dataframe']:
+                # TODO: Move generator expression using that function
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker